pkg/kubelet/cm/container_manager_linux.go

```diff
@@ -54,8 +55,15 @@ const (
 	// The minimum memory limit allocated to docker container: 150Mi
 	MinDockerMemoryLimit = 150 * 1024 * 1024
 
-	dockerProcessName = "docker"
-	dockerPidFile     = "/var/run/docker.pid"
+	dockerProcessName     = "docker"
+	dockerPidFile         = "/var/run/docker.pid"
+	containerdProcessName = "docker-containerd"
+	containerdPidFile     = "/run/docker/libcontainerd/docker-containerd.pid"
```

Comment:
I can't find any documentation around this path, but we do have a fallback (`pidof`) if the file isn't found. I'll follow up with a docker issue to check the stability.